### DISCUSSION TITLE
Problems with low energy radar sensor LD2410s

### DISCUSSION THREAD

**Discussion by peterkowalewski** [arendst/Tasmota#22258] [Support request] [answered]
Hello everyone,
I tried to get the LD2410s radar sensor working with tasmota, but unfortunately, I wasn’t successful. It works very well with the Hi-Link test tool.  The baud rate have be set to 115200. It is not working with 256000. Does anyone have any ideas?

**Comment by sfromis** [comment]
As things are now, the bit rate is hardcoded in the driver, which is thereby where you can modify it.
https://github.com/arendst/Tasmota/blob/f8fbe153000591727e40b5007e0de78c33833131/tasmota/tasmota_xsns_sensor/xsns_102_ld2410.ino#L403-L418

**Reply by peterkowalewski** [reply]
I changed this to 115200, but is still not working. Any idea how to properly log the content of receive buffer:

**Comment by peterkowalewski** [comment, marked as answer]
There is i pull request:
add xsns_102_ld2410s.ino #22253
this will solve my issue.
The current implementation is not supporting LD2410S due to different output data format to the LD2410 device.
